client/src/components/WorkflowInvocationState/WorkflowInvocationState.vue

```diff
@@ -272,6 +274,17 @@ function cancelWorkflowSchedulingLocal() {
                         :loading="!invocationAndJobTerminal"
                         class="jobs-progress" />
                 </div>
+                <BButton
+                    v-if="!invocationAndJobTerminal"
+                    v-b-tooltip.noninteractive.hover
+                    title="Cancel scheduling of workflow invocation"
+                    data-description="cancel invocation button"
+                    size="sm"
+                    pill
+                    variant="outline-danger"
```

Comment:
I'd suggest "downgrading" the visual language. The operation is not dangerous, you lose nothing.
```suggestion
                    variant="outline-warning"
```

Reply:
That would be this:
| Current `outline-danger` |
| ---- |
| ![firefox_zsCj3vgJBD](https://github.com/user-attachments/assets/7e0e60b6-dfe6-4d87-bebc-7fe67e5a3a90) |



| Changed to `outline-warning` |
| ---- |
| ![firefox_DusLmqgIYP](https://github.com/user-attachments/assets/55018684-a741-4402-b67b-01e4d3b0abad) |

Reply:
Both of these don't match our style, also that button is way too big. Have you considered making it a normal stop button next to the play button ?

Reply:
![image](https://github.com/user-attachments/assets/f34125ad-52e4-4273-9fc1-f56dd8d83fa0)

What about this?
The icon can be kept as the ✖️ instead, and the text can be `Cancel` instead of `Stop`
whatever works...

Reply:
Nice, I like it!